website/docusaurus.config.js

```diff
@@ -24,7 +24,7 @@ const config = {
   // If you aren't using GitHub pages, you don't need these.
   organizationName: 'facebookexperimental', // Usually your GitHub org/user name.
   projectName: 'object-introspection', // Usually your repo name.
-	deploymentBranch: 'web',
+	deploymentBranch: 'gh-pages',
```

Comment:
Note that this isn't actually important since you aren't using docusaurus directly to deploy (`yarn deploy`) but good to be correct.